src/pages/organizations/pages/organizations-page/organization-category-list/index.tsx

```diff
@@ -71,16 +71,24 @@ const OrganizationCategoriesList: FC<Props> = ({
             <SC.CategoryBox key={id} className='col-12'>
               <SC.Title>{translate(prefLabel) || name}</SC.Title>
               <SC.Info>
-                <SC.CountTag type={Entity.DATASET}>{datasetCount}</SC.CountTag>
+                <SC.CountTag type={Entity.DATASET} zero={datasetCount <= 0}>
+                  {datasetCount}
+                </SC.CountTag>
                 {!isTransportportal && (
                   <>
-                    <SC.CountTag type={Entity.DATA_SERVICE}>
+                    <SC.CountTag
+                      type={Entity.DATA_SERVICE}
+                      zero={dataserviceCount <= 0}
+                    >
                       {dataserviceCount}
                     </SC.CountTag>
-                    <SC.CountTag type={Entity.CONCEPT}>
+                    <SC.CountTag type={Entity.CONCEPT} zero={conceptCount <= 0}>
```

Comment:
Kan det faktisk være mindre enn 0?

Reply:
Vel, i utgangspunktet ikke, men in case feilverdier. Tenkte det var mer feilsikkert.

Reply:
Fordi hvis det er fare for at det er `undefined` så er det også mulig å ha `===` og bruke && for å sjekke om conceptCount er undefined:
```suggestion
                    <SC.CountTag type={Entity.CONCEPT} zero={conceptCount && conceptCount === 0}>
```